<launch>	
	<node pkg="beginner_tutorials" type="bhand_notice_position" name="bhand_notice_position" output="screen">	
	</node>	
	<!--
	<node pkg="beginner_tutorials" type="pcan_test" name="PCAN_Test" output="screen">
	<param name="dev_name" type="string" value="/dev/pcanusb2" />
	<param name="fifo_send_on" type="bool" value="true" />	
	<param name="display_on" type="bool" value="false" />	
	</node>	
	-->
</launch>
